test/test_core.py

```diff
@@ -5716,6 +5716,7 @@ def test_getdents64(self):
   def test_getdents64_special_cases(self):
     self.do_run_in_out_file_test('fs/test_getdents64_special_cases.cpp')
 
+  @also_with_wasmfs
```

Comment:
This test does not use the JS API, so it is not testing the new functionality here.